### ISSUE TITLE
[UI Update] Create Wallet/Secret Recovery Phrase

### ISSUE THREAD

**Issue by kanniec826** [forbole/forbole-x-mobile#125]

**Comment by Moistbobo** [comment]
For the headers (see screenshot), I would like to leave them default for now as it can get a bit tedious to keep consistent for ios and android:

![image.png](https://images.zenhubusercontent.com/61932bae652159e7c5c78d84/09603e54-e789-460d-bb85-1a2fb0ce024c)

**Comment by Moistbobo** [comment]
Also @Rooty2020  I need to mention that the android version is using the default android system font (I think its Roboto), which doesn't seem to have a semi-bold version, so text that appears semi-bold on the ios version will appear as regular width.

**Comment by kanniec826** [comment]
@Moistbobo So apart from no.4, others can be changed?

**Comment by Moistbobo** [comment]
@kanniec826 yes, actually I need some clarification for no.1; should the `Please write down and store...` text be size 24? Or does that include the title (`Secret Recovery Phrase`) as well?

**Comment by kanniec826** [comment]
@Moistbobo Not include the title (Secret Recovery Phrase)

**Comment by Rooty2020** [comment]
@Moistbobo please use medium font on Android device instead of semi-bold font on iOS device
